src/precompile.jl

```diff
@@ -59,6 +59,8 @@ function _precompile_()
     end
     # traits
     for T in eltypes, C in pctypes
+        @assert precompile(Tuple{typeof(eltype),Type{C}})
+        @assert precompile(Tuple{typeof(eltype),Type{C{T}}})
         @assert precompile(Tuple{typeof(alpha),C{T}})
         C <: AbstractGray && @assert precompile(Tuple{typeof(gray),C{T}})
         C <: AbstractRGB || continue
```

Comment:
This is not directly related to this PR, but are the `red()`, `green()` and `blue()` for `TransparentRGB` not precompiled?

Reply:
No, I could do that but I think transparent colors are less commonly-used. And of course having them for `RGB` saves some time when it does go to compile them.